packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -159,8 +159,9 @@ describe('routes/Product', () => {
         .get('/v1/oauth/subscriptions/customer')
         .reply(200, MOCK_CUSTOMER, { 'Access-Control-Allow-Origin': '*' }),
     ];
-    const { findByTestId } = render(<Subject />);
+    const { findByTestId, queryByTestId } = render(<Subject />);
     await findByTestId('error-loading-profile');
+    expect(queryByTestId('error-loading-profile')).toBeInTheDocument();
```

Comment:
Do we need line 163 then?

Reply:
The linter was complaining about the lack of `expect` on these tests. In this case we do need line `163`, we need to wait until it is in the DOM before we can test for it.

Reply:
Or assign the element found on line 163 into a variable and use it.  There's no need to find/query multiple times.

Reply:
(Sorry I didn't see your response when I wrote my second comment...I thought GH UI is better than that)

So, a) await the rendering, or b) await the expect, or c) assign the element found by `findByTestId` and use it in the assertion.